src/webgl/program.js

```diff
@@ -340,6 +340,15 @@ export default class Program extends Resource {
     return this.gl.getUniform(this.handle, location);
   }
 
+  /* eslint-disable max-len */
+  // Rretrieves information about active uniforms identifed by their indices (`uniformIndices`)
+  // For valid `pname` values check :
+  // https://developer.mozilla.org/en-US/docs/Web/API/WebGL2RenderingContext/getActiveUniforms
+  getActiveUniforms(uniformIndices, pname) {
```

Comment:
Should we at least mention these new methods in the docs?

Reply:
Added to docs.